app/models/assignment.rb

```diff
@@ -435,7 +435,7 @@ def clone_groupings_from(assessment_id)
             unless grouping.memberships << membership # this saves the membership as a side effect, i.e. can return false
```

Comment:
should be `m.role_id` on line 432 (above, sorry can't comment directly on that line)